code/__HELPERS/global_lists.dm

```diff
@@ -54,6 +54,7 @@ GLOBAL_LIST_EMPTY(randomized_pill_icons)
 		var/datum/species/S = new T
 		S.race_key = rkey //Used in mob icon caching.
 		GLOB.all_species[S.name] = S
+		GLOB.roundstart_species[S.name] = S
```

Comment:
make a var on species or a way to see if theyre joinable roundstart.
```suggestion
		if(S.joinable_roundstart)
			GLOB.roundstart_species[S.name] = S
```
in example.

Reply:
otherwise this is just putting all the species in it like all_species.